gravitee-apim-console-webui/src/management/api-score/api-score-dashboard/api-score-dashboard.component.html

```diff
@@ -40,41 +40,64 @@ <h1 class="page-title">API Score</h1>
     <div class="separator-hr"></div>
 
     <mat-card-content class="overview-cards">
-      <mat-card class="card">
+      <mat-card class="card" data-testid="overview-score">
         <mat-card-header>
           <mat-card-title class="card__title">
-            @if (overviewData.overviewScore.averageScore > 70) {
+            @if (averageScore >= 80) {
```

Comment:
The backend send the score using a double value: 0.8 instead of 80.

Reply:
changed